contrib/Findsodium.cmake

```diff
@@ -53,6 +52,9 @@ if (UNIX)
     endif()
 
     if(sodium_USE_STATIC_LIBS)
+        # pkgconfig for libsodium doesn't provide
+        # static lib info, so override PKG_STATIC here..
+        set(sodium_PKG_STATIC_LIBRARIES libsodium.a)
```

Comment:
I would like to see this guarded just in case this changes in the future:
```cmake
if (NOT DEFINED sodium_PKG_STATIC_LIBRARIES)
    ...
endif()
```

Reply:
good suggestion - I will add that.